pkg/middlewares/aws_instance_block_device_test.go

```diff
@@ -118,12 +123,21 @@ func TestAwsInstanceBlockDeviceResourceMapper_Execute(t *testing.T) {
 					},
 				},
 			},
+			func(factory *terraform.MockResourceFactory) {
+				factory.On("CreateResource", mock.Anything, "aws_ebs_volume").Times(2).Return(nil, nil)
```

Comment:
Just FYI, you can use here `Twice()` instead of `Times(2)` but no need to change it.

Reply:
Yeah I know but I prefer to always use `Times`

Reply:
If i == 1 you use `Once` and if i > 1 `Times(i)` ?

Reply:
Yeah and I was wondering myself if I should use `Times` everywhere

Reply:
Indeed, it is more explicit.